modules/webclient/templates/i2b2_config_data.js.erb

```diff
@@ -0,0 +1,19 @@
+{
+	urlProxy: "index.php",
+	urlFramework: "js-i2b2/",
+	//-------------------------------------------------------------------------------------------
+	// THESE ARE ALL THE DOMAINS A USER CAN LOGIN TO
+	lstDomains: 
+	[
+		<% @domains.each do |domain| -%>
+		{
+			domain:        "<%= domain['domain'] %>",
+			name:          "<%= domain['name'] %>",
+			urlCellPM:     "<%= domain['urlCellPM'] %>",
+			allowAnalysis: "<%= domain['allowAnalysis'] %>",
+			debug:         "<%= domain['debug'] %>"
+		},
+		<% end -%>
```

Comment:
Look at ::params. THere's only one domain, and it's settings are there. You can leave debug as a class parameter for `i2b2::webclient_root` defaulting to true. AFAIK, all it does is allow you to look at the messages with the server.